### PR TITLE
DrivAerML: Fourier+no-EMA — AdamW LR sweep (3e-4, 8e-4, 1e-3)

### PR DESCRIPTION
**Pull request by tcapelle**
## Hypothesis
The current best DrivAerML result uses AdamW lr=5e-4 with Fourier features. Without Fourier, lr=8e-4 was optimal (PR #2467). The Fourier features change the gradient landscape — different LR values may be optimal. Testing lr=3e-4 (more conservative), lr=8e-4, and lr=1e-3 (more aggressive) alongside the current lr=5e-4 will identify whether 5e-4 is truly optimal or if we can squeeze more performance.

## Instructions

Run two parallel jobs on both GPUs, then report all results:

**Run 1 — lr=8e-4:**
```bash
cd target/icml2026 && CUDA_VISIBLE_DEVICES=0 python train.py \
  --dataset drivaerml \
  --optimizer adamw \
  --lr 8e-4 \
  --cosine_t_max 30 \
  --no-use-ema \
  --enable-fourier \
  --epochs 999 \
  --batch-size 1 \
  --drivaerml-train-surface-points 50000 \
  --drivaerml-eval-surface-points 50000 \
  --max-train-batches 394 \
  --max-eval-batches 200 \
  --wandb_group gen-drivaerml-lr-sweep &
```

**Run 2 — lr=1e-3:**
```bash
cd target/icml2026 && CUDA_VISIBLE_DEVICES=1 python train.py \
  --dataset drivaerml \
  --optimizer adamw \
  --lr 1e-3 \
  --cosine_t_max 30 \
  --no-use-ema \
  --enable-fourier \
  --epochs 999 \
  --batch-size 1 \
  --drivaerml-train-surface-points 50000 \
  --drivaerml-eval-surface-points 50000 \
  --max-train-batches 394 \
  --max-eval-batches 200 \
  --wandb_group gen-drivaerml-lr-sweep
```

Wait for both to complete. Report `val_primary/surface_rel_l2_pct` for each run, epochs completed, and W&B run IDs. Note which LR performs best.

**Critical:** You must pass `--epochs 999` — the default is only 2 epochs due to a hardcoded bug in the training script.

## Baseline
- **Primary metric:** `val_primary/surface_rel_l2_pct` (lower is better)
- **Current best (val):** 51.35% at lr=5e-4 (PR #2475, W&B run: 5ncrjm32)
- **Current best (test):** 52.06%
- **Prior no-Fourier best:** 56.91% at lr=8e-4 (PR #2467)
- **Reproduce baseline:** `cd target/icml2026 && python train.py --dataset drivaerml --optimizer adamw --lr 5e-4 --cosine_t_max 30 --no-use-ema --enable-fourier --epochs 999 --batch-size 1 --drivaerml-train-surface-points 50000 --drivaerml-eval-surface-points 50000 --max-train-batches 394 --max-eval-batches 200`

### CODE REVIEW

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2526 because Duplicate assignment — closing extra PR created by parallel agent. Student already has a primary experiment assigned..

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2526 because Closing duplicate — student gen already has active PR #2516 assigned..
